app/src/context/evm/prompt-wars-market-contract/PromptWarsMarketContractContextController.tsx

```diff
@@ -126,16 +127,16 @@ export const PromptWarsMarketContractContextController = ({
         isExpiredUnresolved,
         isBeforeMarketEnds,
       ] = await Promise.all([
-        contract.get_market_data(),
-        contract.get_resolution_data(),
-        contract.get_fees_data(),
-        contract.get_management_data(),
-        contract.get_collateral_token_data(),
-        contract.is_resolved(),
-        contract.is_reveal_window_expired(),
-        contract.is_resolution_window_expired(),
-        contract.is_expired_unresolved(),
-        contract.is_before_market_ends(),
+        contract.read.get_market_data(),
+        contract.read.get_resolution_data(),
+        contract.read.get_fees_data(),
+        contract.read.get_management_data(),
+        contract.read.get_collateral_token_data(),
+        contract.read.is_resolved(),
+        contract.read.is_reveal_window_expired(),
+        contract.read.is_resolution_window_expired(),
+        contract.read.is_expired_unresolved(),
+        contract.read.is_before_market_ends(),
       ]);
```

Comment:
@Udit-takkar I managed to fetch the values from our Optimism Sepolia contract without the need of `signer`. Let's proceed with this way of fetching the values. Next step would be to fix the errors produced in the commented components of `PromptWars.tsx`.